include/beast/http/impl/basic_headers.ipp

```diff
@@ -205,6 +206,18 @@ basic_headers(FwdIt first, FwdIt last)
 }
 
 template<class Allocator>
+std::size_t
+basic_headers<Allocator>::
+count(boost::string_ref const& name) const
+{
+    auto const it = set_.find(name, less{});
+    if(it == set_.end())
+        return 0;
+    auto const last = set_.upper_bound(name, less{});
+    return static_cast<std::size_t>(std::distance(it, last));
+}
+
+template<class Allocator>
 auto
 basic_headers<Allocator>::
 find(boost::string_ref const& name) const ->
```

Comment:
Sorry, yup, it's clearly good and expected.  The documentation in the *.hpp file says so.